DockerfileUbuntu

```diff
@@ -7,7 +7,7 @@ COPY . /buildenv
 WORKDIR /buildenv
 
 RUN curl -f https://raw.githubusercontent.com/toniebox-reverse-engineering/tonies-json/release/tonies.json -o /buildenv/install/pre/config/tonies.json || true
-RUN make zip
```

Comment:
I would prefer building the "general" zip, so I can reuse it for releases, without building everything a second time.

Reply:
I don't think this was deliberate. I'll change it accordingly

Reply:
Oh I see.

Reply:
I'll just leave this part as it has been then